pagefind_ui/default/svelte/ui.svelte

```diff
@@ -223,12 +222,12 @@
                     input_el.blur();
                 }
                 if (e.key === "Enter") {
-                    e.preventDefault();
+                     e.preventDefault();
```

Comment:
Extra space, but it's a *really* minor thing.

```suggestion
                    e.preventDefault();
```